backendCore/dl/dl_model_parser.py

```diff
@@ -0,0 +1,84 @@
+from sklearn.tree import *
+from sklearn.ensemble import *
+from sklearn.linear_model import *
+from sklearn.svm import *
+from sklearn.naive_bayes import *
+import torch
+import torch.nn as nn
+from torch.autograd import Variable
+import torchvision.transforms as transforms
+
+
+def parse_deep_user_architecture(user_model):
```

Comment:
Will this parser function be needed in `backend/dl/dl_model_parser` be used? If so, we should remove any duplicate code if we have a new version of this function and it lives strictly under `backendCore/`. We might also want to update this [unit test](https://github.com/DSGT-DLP/Deep-Learning-Playground/blob/nextjs/tests/test_model_parser.py)